### ISSUE TITLE
Accessibility Issue - Duplicate Table Header Announcements for Screen Readers

### ISSUE THREAD

**Issue by sherenemorris03** [DFE-Digital/find-information-products-services-frontend#117]
**Describe the bug**
Issue 9 in the Accessibility report
The ‘Phase’ and ‘Group’ cells in the product description table include visually hidden text that repeats the table header. Since screen readers already announce the header associated with each cell, this results in users hearing “Phase, Phase” or “Group, Group,” which can be confusing and disrupt comprehension.
Additionally, when table cells are empty, users are not informed that the cell is blank, which may lead to misinterpretation of missing data.

User story:
As a screen reader user,
I want to hear table headers only once when navigating cells,
So that I can understand the table content without confusion or repetition.

**To Reproduce**
1. Navigate to the Product description page.
2. Use a screen reader to explore the table.
3. Observe the repeated header names and lack of indication for empty cells.

**Expected behaviour**
Table headers should be announced once per cell. Empty cells should be clearly indicated to screen reader users.
Headers are announced twice due to redundant hidden text. Empty cells are silent, offering no indication of missing data.

**Screenshots**

<img width="946" height="291" alt="Image" src="https://github.com/user-attachments/assets/afebc3e6-00d7-4f57-8aff-024227b999e1" />

<img width="812" height="266" alt="Image" src="https://github.com/user-attachments/assets/b5212e9f-8308-45ed-9295-be5724fd673b" />

**Additional context**
The additional hidden content can be removed. Also, when table data is empty such as this example, users should be made aware that it is blank. 

Link to issue on Statement Service:
https://accessibility-statements.education.gov.uk/services/DACFEE24-0602-46AD-AE7C-19A23DADA42D/issues/0CE86227-C0F3-4116-9E9E-486D1C8A1571
